src/sidekick/fr.css

```
@@ -65,3 +65,11 @@
 .hlx-sk .unpublish button::before {
   content: "Dépublier";
 }
+
+.hlx-sk-special-view .description::before {
+  content: "Rendu de données convivial."
```

ah ah, nice Google translation :) But now that this is written, I have hard time to find something you would write in french...
```suggestion
  content: "Aperçu des données."
```